docs/examples/http/server.py

```diff
@@ -39,19 +39,17 @@
 else:
     exporter = ConsoleSpanExporter()
 
-# The preferred tracer implementation must be set, as the opentelemetry-api
-# defines the interface with a no-op implementation.
-trace.set_preferred_tracer_provider_implementation(lambda T: TracerProvider())
 tracer = trace.get_tracer(__name__)
 
 # SpanExporter receives the spans and send them to the target location.
 span_processor = BatchExportSpanProcessor(exporter)
-trace.tracer_provider().add_span_processor(span_processor)
+trace.set_tracer_provider(TracerProvider())
```

Comment:
This should be done before L42, otherwise the returned tracer will be de default one. I pushed a fix as a part of https://github.com/open-telemetry/opentelemetry-python/pull/483/commits/a747b61374ec8f6d5d30335c68f24c495c726d21.